applications/subxt-python.md

```diff
@@ -0,0 +1,196 @@
+  - Researchers and data analysts who want to query and analyze data from Substrate-based networks using Python.
+  - Existing Python-based projects or organizations looking to integrate with Substrate/Polkadot/Kusama ecosystems.
+- What need(s) does the project meet?
+  - Provides a Python interface for interacting with Substrate-based blockchains, enabling Python developers to build
+  and integrate applications easily.
+  - Allows Python developers to leverage the extensive Python ecosystem and libraries for data analysis, machine
+  learning, and other use cases in conjunction with Substrate-based blockchains.
+  - Facilitates the adoption and growth of the Substrate/Polkadot/Kusama ecosystem by attracting Python developers and
+  expanding the developer base. Existing Substrate teams could leverage Python developers to build their projects. 
+- How did we identify these needs? Please provide evidence in the form of (scientific) articles, forum discussions, case studies, or raw data.
+  - An analysis of the Substrate/Polkadot/Kusama ecosystem reveals a lack of mature and actively maintained Python
+  libraries for interacting with Substrate-based blockchains, indicating a gap that `subxt-python` aims to fill.
+  - There's about 104 questions on [substrate stackexchange](https://substrate.stackexchange.com/search?q=python) 
+    signaling the need for Python support in the Substrate ecosystem.
+- Are there any other projects similar to yours in the Substrate / Polkadot / Kusama ecosystem?
+  - Yes, there is a project called [py-substrate-interface](https://github.com/polkascan/py-substrate-interface) which
+  is a Python library for interacting with Substrate-based blockchains. `subxt-python` is built on top of the subxt Rust
+  library, leveraging its features, performance, and type-safe API. It provides a one-to-one mapping of `subxt`'s
+  functionality to Python allowing developers to quickly iterate on their projects. `subxt-python` benefits from the
+  active development and maintenance of `subxt`, ensuring compatibility with the latest Substrate versions and features.
+- Are there any projects similar to yours in related ecosystems?
+  - In the Ethereum ecosystem, there are Python libraries like [web3.py](https://web3py.readthedocs.io/en/stable/) and
+  [eth-brownie](https://github.com/eth-brownie/brownie) that provide similar functionality for interacting with
+  Ethereum-based blockchains. These libraries have gained significant adoption and have proven to be valuable tools for
+  Python developers working with Ethereum.
+
+## Team
+### Team members
+
+- Name of team leader: Tuguldur Baigalmaa
+- Names of team members: Tuguldur Baigalmaa
+
+### Contact
+
+- **Contact Name:** Tuguldur Baigalmaa
+- **Contact Email:** tgldr0511@gmail.com
+- **Website:** https://philoniare.com/about
+
+### Legal Structure
+
+- **Registered Address:** None
+- **Registered Legal Entity:** None
```

Comment:
You listed a team name at the top, but no legal entity here. If there is more than one beneficiary, please add a legal entity or change the team name to your own.

Reply:
Ah makes sense, updated team name